lib/kitchen/directions/bake_injected_exercise/bake_injected_exercise.rb

```diff
@@ -1,17 +1,71 @@
 # frozen_string_literal: true
 
 module Kitchen::Directions::BakeInjectedExercise
-  def self.v1(exercise:)
-    V1.new.bake(exercise: exercise)
+  def self.v1(exercise:, alphabetical_multiparts: false)
+    V1.new.bake(exercise: exercise, alphabetical_multiparts: alphabetical_multiparts)
```

Comment:
Using an argument passed in to determine whether this gets applied to questions with question-stimulus in injected exercises.

Reply:
this would be a good opportunity for an `options` arg (see: https://github.com/openstax/cnx-recipes/issues/4701)